frontend/src/components/Address.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import { GeoapifyGeocoderAutocomplete, GeoapifyContext } from '@geoapify/react-geocoder-autocomplete'
+import '@geoapify/geocoder-autocomplete/styles/minimal.css'
+
+const Address = () => {
+
+  const geoapifyKey = process.env.REACT_APP_GEOAPIFY_KEY
+
+  function onPlaceSelect(value) {
+    console.log(value);
+  }
+
+  function onSuggectionChange(value) {
+    console.log(value);
+  }
```

Comment:
Depending on the component provided by geoapi, you probably don't get the value right away do you? There is probably an event fired which contains the value and maybe some methods/other values. 
Just fyi if it does: I would suggest naming it event and probably call `event.preventDefault()`  before adding your event/value handling below (this applies only if it is really an event that gets passed and not just a value).

Reply:
I will check this when I will start with the backend side. Thanks for the suggestion ;)